tests/test_null.py

```diff
@@ -3,23 +3,70 @@
 import pytest
 from preprocessy.handlenullvalues import NullValuesHandler
 
+dataframe = pd.read_csv("datasets/encoding/test2.csv")
+
 
 class TestHandlingNullValues:
     def test_null_dataframe(self):
         with pytest.raises(ValueError):
             handler = NullValuesHandler()
             handler.execute()
 
-    def test_incorrect_input_type(self):
-        self.dataframe = pd.read_csv("datasets/encoding/testnew.csv")
-        with pytest.raises(TypeError):
-            handler = NullValuesHandler(df=self.dataframe, drop="nice")
+    def test_none_args(self):
+        with pytest.raises(Exception):
+            handler = NullValuesHandler(df=dataframe)
             handler.execute()
 
     def test_multiple_args(self):
-        self.dataframe = pd.read_csv("datasets/encoding/testnew.csv")
+        with pytest.raises(Exception):
+            handler = NullValuesHandler(df=dataframe, drop=True, fill_missing="mean")
+            handler.execute()
+
+        with pytest.raises(Exception):
+            handler = NullValuesHandler(
+                df=dataframe, drop=True, fill_values={"Test": "Tata"}
+            )
+            handler.execute()
+
         with pytest.raises(Exception):
             handler = NullValuesHandler(
-                df=self.dataframe, drop=True, fill_missing="mean"
+                df=dataframe, fill_missing="mean", fill_values={"Test": "Tata"}
             )
             handler.execute()
+
+    def test_incorrect_input_type(self):
+        # for dataframe argument
+        array = np.random.random((5, 5))
+        with pytest.raises(TypeError):
+            handler = NullValuesHandler(df=array)
+            handler.execute()
+
+        # for drop argument
+        with pytest.raises(TypeError):
+            handler = NullValuesHandler(df=dataframe, drop="nice")
+            handler.execute()
+
+        # for fill_missing argument
+        with pytest.raises(TypeError):
+            handler = NullValuesHandler(df=dataframe, fill_missing=3)
+            handler.execute()
+
+        with pytest.raises(Exception):
+            handler = NullValuesHandler(df=dataframe, fill_missing="sum")
+            handler.execute()
+
+        # for fill_values argument
+        with pytest.raises(TypeError):
+            value = [5]
+            handler = NullValuesHandler(df=dataframe, fill_values=value)
+            handler.execute()
+
+        with pytest.raises(Exception):
+            value = {"Label": 10}
+            handler = NullValuesHandler(df=dataframe, fill_values=value)
+            handler.execute()
+
+    def test_output(self):
```

Comment:
Can you add a few more test cases here? Something to test whether correct values are being filled here.

Reply:
Sure, will add more test cases